### PR TITLE
Updated .gitignore

### LINKED ISSUE #183: Add pytest cache to gitigore
🐞 **Problem**
`.pytest_cache` is generated automatically sometimes. It may show up in a PR.

🎯 **Goal**
  - Add it to `.gitignore`

💡 **Possible solutions**
  - Add `*/.pytest_cache` to `.gitignore` under proper heading. Create a new section if an appropriate one is not found.

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #183`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by Manvi07**
Fixes #183

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -65,3 +65,6 @@ pip-wheel-metadata
 
 #jupyter notebook
```
Comment (ritzvik) [inline .gitignore]: Could you remove `.ipynb_checkpoints from here. I can see it is already there in other generated files

File: .gitignore [hunk 2]
```diff
@@ -65,3 +65,6 @@ pip-wheel-metadata
...
 *.ipynb_checkpoints
+
+#default pytest cache directory
```
Comment (ritzvik) [inline .gitignore]: Could you add a space after #. I don't know why I do this 😄

**Comment by Manvi07** [comment]
Done :)

**Comment by ritzvik** [comment]
> Done :)

Thanks @Manvi07 . @Masquerade0097 See to it.

**Comment by Masquerade0097** [comment]
Thanks for the contribution.

**Pull request merged**